_data/master/navbars/networking.yml

```diff
@@ -25,7 +25,7 @@ toc:
    - title: Get started with IP address management
      path: /networking/get-started-ip-addresses
    - title: Configure IP autodetection
```

Comment:
The title here doesn't match the title in the doc (auto detection vs autodetection)

Reply:
Good catch. Aligned doc and TOC to "autodetection"